app/connectors/GetIncomeSourceDetailsConnector.scala

```diff
@@ -27,7 +27,7 @@ class GetIncomeSourceDetailsConnector @Inject()(http: HttpClient, val appConfig:
 
   def getIncomeSourceDetails(nino: String, taxYear: String, incomeSourceId: String)(implicit hc: HeaderCarrier): Future[IncomeSourcesDetailsResponse] = {
     val incomeSourcesUrl = appConfig.desBaseUrl +
```

Comment:
does this val need to be incomeSourceUrl to match?

Reply:
aye